unitex/src/fr/umlv/unitex/leximir/helper/DelasHelper.java

```diff
@@ -80,13 +81,16 @@ else if(dic!=null){
             //String path = Utils.getValueXml("pathDelas")+"/"+dela;
             String path="";
             if(allDelas)
-            path = DictionaryPath.allDelas+File.separator+dela;
+            	path = DictionaryPath.allDelas+File.separator+dela;
```

Comment:
All binary operators except `.` should be separated from their operands by spaces